app/connectors/Parser.scala

```diff
@@ -23,6 +23,11 @@ import uk.gov.hmrc.http.HttpResponse
 trait Parser {
 
   protected val parserName: String
+  protected val service : String = "income-tax-property-frontend"
+
+  def logMessage(response:HttpResponse): Option[String] ={
+    Some(s"[$parserName][read] Received ${response.status} from $service API. Body:${response.body}")
+  }
```

Comment:
Return type should be String and call sites should wrap this in a Some if needed